### ISSUE TITLE
Performance Fish seems to cause endless load on startup

### ISSUE THREAD

**Issue by Hyomoto** [bbradson/Performance-Fish#24]
I don't know if this could be related to my modlist in general since if I understand what PF does, these patches should be applied before it ever gets around to loading the modlist.  Nonetheless I've had it hang on this message several times,
![image](https://github.com/bbradson/Performance-Fish/assets/6281477/61f0842c-ccce-4d84-8dc8-784521acefa1)

Of course, I've also had issues with Prepatcher that may be related so I don't know that PF itself is responsible.  What I can say about reproduction is that force closing Rimworld and starting it again will *usually* proceed without issue so its not a consistent error, and that it *appears* to be at least somewhat influenced if after starting Rimworld I click out to do something else while I wait for it to load (my overloaded modlist takes about five to ten minutes).
[Player.log](https://github.com/bbradson/Performance-Fish/files/13789002/Player.log)

The player.log talks about some fallback handlers which is probably significant to someone in the know but a Google search suggests this is some oddity with Unity and not with Rimworld specifically.

**Comment by Hyomoto** [comment]
This *might* have something to do with changing modlists.  I trimmed my list down to just a handful of mods for the sake of writing a patch.
![image](https://github.com/bbradson/Performance-Fish/assets/6281477/133dea49-47fd-4e13-b3f7-d78139b2090a)
And the first time I set up this list, and when I made a change to the location of my patch, both times the hang happened.  So, to test the theory I went ahead and added a mod: freeze happens.  It doesn't seem to be specific to the mod, just that the list changed.
